app/controllers/application_controller/ci_processing.rb

```diff
@@ -909,7 +909,7 @@ def reconfigure_update
         javascript_redirect previous_breadcrumb_url
       end
     when "submit"
-      options = {:src_ids => params[:objectIds]}
+      options = {:request_type => :vm_reconfigure, :src_ids => params[:objectIds]}
```

Comment:
@carbonin - is request_type needed here? make_request is called for VmReconfigureRequest.